import.sql

```diff
@@ -1 +1,8 @@
-\COPY questions(id, product_id, body, date_written, asker_name, asker_email, reported, helpful) FROM '/Users/noahbeito/Desktop/RFP2303/QuestionsAPI/data/questions.csv' DELIMITER ',' CSV HEADER;
\ No newline at end of file
+-- psql command to run this file in terminal:
+-- \i /Users/noahbeito/Desktop/RFP2303/QuestionsAPI/import.sql
+
+\COPY questions(id, product_id, body, date_written, asker_name, asker_email, reported, helpful) FROM '/Users/noahbeito/Desktop/RFP2303/QuestionsAPI/data/questions.csv' DELIMITER ',' CSV HEADER;
```

Comment:
Assuming you're importing everything from the file, I actually don't think you need to add the `(id, product_id, body, date_written, asker_name, asker_email, reported, helpful)`

like for one of mine it's just 
`\COPY products FROM '/home/davidguy/Documents/hack-reactor/SDC/products-api/csv-files/product.csv' DELIMITER ',' CSV HEADER;`